### PR TITLE
SEO-172844-WPF-weblink-validatetor-image

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Please review the changes made in image name alt text and image folder.
Thanks.
Regards,
Edith.

### CODE REVIEW

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Diagram/Getting-Started.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Create new WPF project using Visual Studio. For more [details](https://learn.mic
 
 [WPF Diagram](https://www.syncfusion.com/wpf-controls/diagram) (SfDiagram) control can be added to the application by dragging it from Toolbox and dropping it in Designer view. The required assembly references will be added automatically.
 
-![WPF Diagram from Designer](Getting-Started_images\wpf-diagram-from-designer.png)
+![WPF Diagram from Designer](getting-started_images\wpf-diagram-designer.png)
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply wpf/Diagram/Getting-Started.md:30]: Resolved

File: wpf/Diagram/Getting-Started.md [hunk 2]
```diff
@@ -718,13 +718,13 @@ Declare the style for node, connector, symbol, and symbol group to visualize the
...
 
-![WPF Diagram Stencil Output](Getting-Started_images\wpf-diagram-stencil-output.PNG)
+![WPF Diagram Stencil Output](getting-started_images\wpf-diagram-stencil-output.PNG)
 
 #### Interaction with stencil
 
 We have represented the steps to interact with stencil such as drag and drop elements from stencil to diagram in the following gif.
 
-![WPF Diagram Interact with Stencil](Getting-Started_images\wpf-diagram-interact-with-stencil.gif)
+![WPF Diagram Interact with Stencil](getting-started_images\wpf-diagram-interact-stencil.gif)
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply wpf/Diagram/Getting-Started.md:727]: resolved

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**